client/src/components/hud/info.zig

```diff
@@ -0,0 +1,105 @@
+const common = @import("common.zig");
+const config = @import("../../config.zig");
+const rl = @import("raylib");
+const std = @import("std");
+const GameState = @import("../../game/state.zig");
+
+const barHeight = 40;
+
+fn stats(character: GameState.World.Character, size: rl.Vector2, position: rl.Vector2, fontSize: i32, allocator: std.mem.Allocator) !void {
+    const nameLength: f32 = @floatFromInt(rl.measureText(character.stats.name, fontSize));
+
+    var index: [4]u8 = undefined;
+    const levelNumberStr = std.fmt.bufPrintZ(index[0..], "{d}", .{character.stats.level}) catch unreachable;
+    const levelStr = try std.fmt.allocPrintZ(allocator, "{s}{s}", .{ "Lvl. ", levelNumberStr });
```

Comment:
Do we really need to allocate these every frame for every character? And if we do, why are we not freeing them?

Reply:
I'll free it, because I like keeping things local to their use case.